test/system/dashboard_test.rb

```diff
@@ -0,0 +1,27 @@
+require "application_system_test_case"
+
+class DashboardSystemTest < ApplicationSystemTestCase
```

Comment:
Shouldn't this just be `DashboardTest` to match the filename?

Reply:
I had problems with some system test previously that had the same name as some other test. I did the same thing as https://github.com/openstreetmap/openstreetmap-website/commit/45e6ebcafc5741750357b7fd6e11ec96c2cbfb24.

Reply:
We have test named after:
- a thing in singular
- a thing in plural
- an action as a noun
- an action as a verb

In the [guides](https://guides.rubyonrails.org/v7.1/testing.html#system-testing) they have system tests named after things in plural and model tests after things in singular. If we want to do the same, we'll have to rename, for example, `ConfirmationResendSystemTest` to `ConfirmationsTest`.